src/common/mem.c

```diff
@@ -84,10 +85,10 @@ umempobj_settings_init(bool md_on_ssd)
 	};
 
 	d_getenv_uint("DAOS_MD_DISABLE_BMEM_V2", &md_disable_bmem_v2);
```

Comment:
Given that V2 format is default now, I think this DAOS_MD_DISABLE_BMEM_V2 is redundant now (This env var and the umempobj_allow_md_bmem_v2() could be removed). What do you think?

Reply:
Sure. I have removed this tunable.